spec/latest/core/index.html

```diff
@@ -723,11 +723,14 @@
       is to give the client a template to construct such a URL at runtime.
       In Hydra, the <i>IriTemplate</i> class is used to do so.</p>
 
-    <p>A <i>IriTemplate</i> consists of a <i>template</i>, whose syntax is
-      specified in [[!RFC6570]], and a set of <i>mappings</i>. Each
-      <i>IriTemplateMapping</i> maps a <i>variable</i> used in the template
+    <p>An <i>IriTemplate</i> consists of a <i>template</i> literal
+      and a set of <i>mappings</i>.
```

Comment:
The line gets too long if I do; similar line wrapping is applied on the rest of the file.